python/gui/auto_generated/processing/qgsprocessingfavoritealgorithmlog.sip.in

```diff
@@ -0,0 +1,91 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/gui/processing/qgsprocessingfavoritealgorithmlog.h               *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
+ ************************************************************************/
+
+
+
+
+
+class QgsProcessingFavoriteAlgorithmLog : QObject
```

Comment:
```suggestion
class QgsProcessingFavoriteAlgorithmLog : QObject
```

Could we change "log" to "manager"? I think this is closer to eg the bookmark manager vs a log (which implies some running tracking of actions)

Reply:
Renamed.